apps/api/src/domain/models/bibliographic-reference/book-bibliographic-reference/commands/create-book-bibliographic-reference/create-book-bibliographic-reference.command.ts

```diff
@@ -0,0 +1,66 @@
+import { Command } from '@coscrad/commands';
+import {
+    ISBN,
+    NestedDataType,
+    NonEmptyString,
+    PositiveInteger,
+    RawDataObject,
+    URL,
+    UUID,
+    Year,
+} from '@coscrad/data-types';
+import { IsNonEmptyArray } from '@coscrad/validation';
+import { AggregateId } from '../../../../../types/AggregateId';
+import { ICreateCommand } from '../../../../shared/command-handlers/interfaces/create-command.interface';
+import BibliographicReferenceCreator from '../../../common/bibliographic-reference-creator.entity';
+
+// convenient shorthand
+const isOptional = true;
+
+@Command({
+    type: 'CREATE_BOOK_BIBLIOGRAPHIC_REFERENCE',
```

Comment:
We may want to create an enum to manage our command types or maybe even nested enum-like objects?

Reply:
added a [story](https://www.pivotaltracker.com/story/show/183227660)